File: PHP/card.php
<?php
require_once("payment.php");
class Card extends Payment{
    public $number;
    public $cvv;
    public $date;
    public function __construct($id, $number, $cvv, $date){
        parent::__construct($id);
        $this->email = $number;
        $this->cvv = $cvv;
        $this->date = $date;
    }
}

File: Python/card.py
from code import interact
from payment import Payment

class Cash(Payment):
    number = int
    cvv = int
    date = str

    def __init__(self, id, number, cvv, date):
        super().__init__(id)
        self.number = number
        self.cvv = cvv
        self.date = date